api/v1beta2/types.go

```diff
@@ -400,3 +400,53 @@ type VPCEndpoint struct {
 	// +optional
 	LBID *string `json:"loadBalancerIPID,omitempty"`
 }
+
+// GenericResourceReference represents a generic IBM Cloud resource.
+type GenericResourceReference struct {
```

Comment:
I don't see this is used anywhere!

Reply:
Sorry, too early for that resource, I'll remove it for this PR.

Reply:
Removed